orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -26,11 +27,12 @@ def __init__(self, model):
         self._export_mode = torch.onnx.TrainingMode.TRAINING
 
     @staticmethod
-    def execution_session_run_forward(execution_session, onnx_model, device, *inputs):
+    def execution_session_run_forward(execution_session, onnx_model, device, fast_path, *inputs):
```

Comment:
Is it possible to set this as a static setting on `GraphExecutionManager` class and not having to pass this to function signatures (including static methods)?

Reply:
I'd like the setting to be ortmodule specific, so I move the device check logic to forward(), just like we did for backward(). this way I can access self and thus remove redundant function params.